superset-frontend/src/components/AvatarIcon.tsx

```diff
@@ -0,0 +1,61 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React, { PureComponent } from 'react';
+import { Tooltip, OverlayTrigger } from 'react-bootstrap';
+import Avatar, { ConfigProvider } from 'react-avatar';
+
+interface Props {
+  firstName: string;
+  iconSize: string;
+  lastName: string;
+  tableName: string;
+  userName: string;
+}
+
+export default class AvatarIcon extends PureComponent<Props> {
+  render() {
+    const { tableName, firstName, lastName, userName, iconSize } = this.props;
+    const uniqueKey = tableName.concat('_', userName);
+    const fullName = firstName.concat(' ', lastName);
+    const colors = [
```

Comment:
can we pull this array out into a constant so that it doesn't get rebuilt on every render?

Reply:
Curious where those colors came from. We should use the main palette (https://github.com/apache-superset/superset-ui/blob/master/packages/superset-ui-color/src/colorSchemes/categorical/airbnb.ts) for now and eventually the pretty SIP-34 ones.
<img width="1414" alt="Screen Shot 2020-06-11 at 10 08 47 PM" src="https://user-images.githubusercontent.com/487433/84467201-3f61c080-ac30-11ea-8870-28cc2658c66b.png">